mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -158,8 +160,12 @@ CLASS net/minecraft/class_1321 net/minecraft/client/MinecraftClient
 	METHOD method_28829 getToastManager ()Lnet/minecraft/class_5611;
 	METHOD method_28830 getTutorialManager ()Lnet/minecraft/class_6119;
 	METHOD method_28831 initializeGame ()V
+	METHOD method_28834 createWindow ()V
+	METHOD method_28835 setDisplayMode ()V
+	METHOD method_28836 setIcon ()V
 	METHOD method_28837 checkIs64Bit ()Z
 	METHOD method_28838 startTimerHackThread ()V
+	METHOD method_28839 setFullscreenDisplayMode ()V
```

Comment:
Since these don't take arguments `init<Thing>` might be a better fit, but I'm fine with `set`